python/paddle/fluid/tests/unittests/test_optimizer_in_control_flow.py

```diff
@@ -225,5 +227,58 @@ def test_optimzier_in_switch(self):
                                                                    loss_2))
 
 
+class TestMultiOptimizersMultiCardsError(unittest.TestCase):
```

Comment:
You should add this unittest here https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/tests/unittests/CMakeLists.txt#L327 so that it can run in 2 gpu cards in CI.

Reply:
Thanks. Done